cmd/kueuectl/app/list/list_clusterqueue.go

```diff
@@ -0,0 +1,265 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package list
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/cli-runtime/pkg/printers"
+	cliflag "k8s.io/component-base/cli/flag"
+
+	"sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/client-go/clientset/versioned"
+	"sigs.k8s.io/kueue/client-go/clientset/versioned/scheme"
+	kueuev1beta1 "sigs.k8s.io/kueue/client-go/clientset/versioned/typed/kueue/v1beta1"
+)
+
+const (
+	cqShort   = `List ClusterQueues`
+	cqLong    = `Lists all ClusterQueues, potentially limiting output to those that are active/inactive and matching the label selector.`
+	cqExample = `  # List ClusterQueue
+  kueuectl list cq|clusterqueue(s)`
+)
+
+type clusterQueueActive string
```

Comment:
I think it will be useful for all lists.

Reply:
Is it possible to move to *bool as we discussed?

Reply:
> Should I move it to some other place?

Yes, I propose to move it to helpers. Make something like this. Than we can use it on localqueue in future.
```
func addActiveFlagVar(cmd *cobra.Command, p *string) {
	cmd.Flags().StringVar(p, "active", "",
		"Filter by active status of cluster queues. Valid values are '*', 'true', 'false'.")
}
```

And also validation and common functions.

Reply:
Also you can check how dry_run already implemented. I think it would be great to implement in the same way.

Reply:
> I tried, but BoolVar and BoolVarP needs some default value (true or false, null is not allowed)

BoolSlice could work, if len() == o , is nor specified, if len() > 1 validation error.